server/plugins/public.js

```diff
@@ -114,6 +118,9 @@ function register (server, options, next) {
     if (isStorePath) {
       return reply(fs.createReadStream(path.join(storePublicPath, 'index.html')))
     }
+    if (is404 && isHoodiePath) {
+      return reply(fs.createReadStream(app))
+    }
```

Comment:
with the change above, we can do 

```js
if (isHoodiePath) {
  return reply(fs.createReadStream(path.join(hoodiePublicPath, 'index.html')))
}
```

and remove the `if (isHoodiePath) {` block below